src/metadata.rs

```diff
@@ -132,3 +124,28 @@ pub fn read_metadata(path: &Path) -> Result<Metadata, MetadataError> {
         date_time_taken,
     })
 }
+
+#[cfg(test)]
+mod tests {
```

Comment:
Awesome, I didn't know how the tests are done :)

Reply:
this is more of a placeholder test to get an idea how they are done. later on, I'd expect us to agree on what exactly do we want to test